CMakeLists.txt

```diff
@@ -208,14 +208,34 @@ function(determine_version OUTPUT_VARIABLE)
 
   if(VERSION_MATCH)
     set(${OUTPUT_VARIABLE} ${CMAKE_MATCH_1} PARENT_SCOPE)
+    set(VERSION_STRING ${CMAKE_MATCH_1})
   else()
     message(FATAL_ERROR "Invalid tag detected: ${NEW_VERSION}")
   endif()
+
```

Comment:
This whole block could be run as a separate step, right? Could we have a separate function for it instead, and call it only where relevant (i.e. Flutter build, possibly MSI stuff later)?

Reply:
Yes, that is what I meant. Should have been clearer. BTW, I like that you are printing the vars that are getting derived. It would be nice if you could keep that.